sormas-backend/src/main/java/de/symeda/sormas/backend/externaljournal/ExternalJournalService.java

```diff
@@ -185,20 +186,20 @@ private String getPatientDiaryAuthTokenInternal() {
 	/**
 	 * Notify external journals that a followUpUntilDate has been updated
 	 *
-	 * @param contact
-	 *            a contact assigned to a person already available in the external journal
+	 * @param caze
+	 *            a case assigned to a person already available in the external journal
 	 * @param previousFollowUpUntilDate
 	 *            the follow-up end date before the update
 	 */
-	public void notifyExternalJournalFollowUpUntilUpdate(ContactDto contact, Date previousFollowUpUntilDate) {
-		PersonDto person = personFacade.getPersonByUuid(contact.getPerson().getUuid());
+	public void notifyExternalJournalFollowUpUntilUpdate(CaseDataDto caze, Date previousFollowUpUntilDate) {
```

Comment:
It does not really make too much sense to require the whole case dto for getting it's person uuid and followup date.

would be cleaner and more performant to have only the used values as function arguments: personUuid, followupUntilDate and prevFollowupUntilDate